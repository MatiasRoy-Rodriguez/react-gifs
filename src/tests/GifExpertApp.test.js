import React from 'react';
import { shallow } from 'enzyme';
import { GifExpertApp } from '../GifExpertApp'


describe('Prueba en <GifExpertApp />', () => {

	test('Debe mostarse correctamente ', () => {

		const wrapper = shallow(<GifExpertApp />)

		expect(wrapper).toMatchSnapshot();

	})

	test('Debe de mostrar una lista de categorias', () => {
		
		const categories = ['DBZ', 'Movies'];
		const wrapper = shallow(<GifExpertApp defaultCategories={categories} />);

		expect( wrapper ).toMatchSnapshot();
		expect( wrapper.find('GifGird').length).toBe( categories.length );

	})
	

})