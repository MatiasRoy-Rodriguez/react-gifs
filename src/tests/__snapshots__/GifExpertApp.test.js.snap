// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Prueba en <GifExpertApp /> Debe de mostrar una lista de categorias 1`] = `
<Fragment>
  <h2>
    Gif App
  </h2>
  <AddCategory
    setCategories={[Function]}
  />
  <hr />
  <ol>
    <GifGrid
      category="DBZ"
      key="DBZ"
    />
    <GifGrid
      category="Movies"
      key="Movies"
    />
  </ol>
</Fragment>
`;

exports[`Prueba en <GifExpertApp /> Debe mostarse correctamente  1`] = `
<Fragment>
  <h2>
    Gif App
  </h2>
  <AddCategory
    setCategories={[Function]}
  />
  <hr />
  <ol />
</Fragment>
`;
